keycloak_user_export/management/commands/migrate_users_to_keycloak.py

```diff
@@ -0,0 +1,224 @@
+    and no associated social-auth record for the "ol-oidc" provider.
+    Keycloak users are created in the realm defined by the `KEYCLOAK_REALM_NAME`
+    environment variable.
+    The `KEYCLOAK_BASE_URL` environment variable must be defined and equal to the
+    base URL of the Keycloak instance.
+    """
+
+    def add_arguments(self, parser):
+        """parse arguments"""
+
+        # pylint: disable=expression-not-assigned
+        parser.add_argument(
+            "email",
+            help="Email address of a Keycloak realm admin user.",
+        )
+        parser.add_argument(
+            "password",
+            help="Password of a Keycloak realm admin user.",
+        )
+        parser.add_argument(
+            "client_id",
+            help="Client ID for the Keycloak Admin-CLI client.",
+        )
+        parser.add_argument(
+            "client_secret",
+            help="Client secret for the Keycloak Admin-CLI client.",
+        )
+        parser.add_argument(
+            "--batchsize",
+            nargs="?",
+            default=25,
+            type=int,
+            help="(Optional) How many users to export to Keycloak at a time.",
+        )
+
+    def get_access_token(
+        self, client_id: str, email: str, password: str, client_secret: str
+    ):
+        """
+        Creates a new access token for a Keycloak realm administrator for use with the admin-cli client.
+
+        Args:
+            client_id (str): The client_ID associated with Keycloak's admin-cli client.
+            email (str): The email address of a Keycloak realm administrator user.
+            password (str): The password associated with the email address for a Keycloak realm administrator user.
+            client_secret (str): The client secret associated with Keycloak's admin-cli client.
+
+        Returns:
+            A new access_token for the administrator user for use with the Keycloak admin-cli client.
+        """
+        url = f"{settings.KEYCLOAK_BASE_URL}/realms/{settings.KEYCLOAK_REALM_NAME}/protocol/openid-connect/token"
+        payload = f"{urlencode({'client_id': client_id})}&{urlencode({'username': email})}&{urlencode({'password': password})}&grant_type=password&{urlencode({'client_secret': client_secret})}&{urlencode({'scope': 'email openid'})}"
+        headers = {"Content-Type": "application/x-www-form-urlencoded"}
+
+        response = requests.request("POST", url, headers=headers, data=payload)
+        return response.json()["access_token"]
+
+    def handle(self, *args, **kwargs):
+        if (
+            not hasattr(settings, "KEYCLOAK_BASE_URL")
+            or settings.KEYCLOAK_BASE_URL is None
+        ):
+            raise CommandError(
+                "KEYCLOAK_BASE_URL must be defined as an environment variable."
+            )
+
+        if (
+            not hasattr(settings, "KEYCLOAK_REALM_NAME")
+            or settings.KEYCLOAK_REALM_NAME is None
```

Comment:
You can simplify this and similar checks to:
```suggestion
            getattr(settings, "KEYCLOAK_REALM_NAME", None) is None
```